### PR TITLE
Add manifest for feedback stack

### PR DESCRIPTION
**Pull request by dhee-tree**
Adding manifest for
- dp-feedback-api
- dp-frontend-feedback-controller

### CODE REVIEW

File: v2/manifests/core-ons/dp-feedback-api.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    x-repo-url: "https://github.com/ONSdigital/dp-feedback-api"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-feedback-api
+      dockerfile: Dockerfile.local
+    command:
+      - reflex
+      - -d
+      - none
+      - -c
+      - ./reflex
```
Comment (lindenmckenzie) [inline v2/manifests/core-ons/dp-feedback-api.yml:12]: dp-feedback-api doesn't have a reflex file.

File: v2/manifests/core-ons/dp-feedback-api.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    environment:
+      BIND_ADDR:                          ":28600"
+      GRACEFUL_SHUTDOWN_TIMEOUT:          ${GRACEFUL_SHUTDOWN_TIMEOUT:-5s}
+      HEALTHCHECK_INTERVAL:               ${HEALTHCHECK_INTERVAL:-30s}
+      HEALTHCHECK_CRITICAL_TIMEOUT:       ${HEALTHCHECK_CRITICAL_TIMEOUT:-90s}
+    healthcheck:
+      test: ["CMD", "curl", "-sSf", "http://localhost:28600/health"]
+      interval: ${HEALTHCHECK_INTERVAL:-30s}
+      timeout: 10s
+      retries: 10
```
Comment (lindenmckenzie) [inline v2/manifests/core-ons/dp-feedback-api.yml:29]: New line at EOF please - you can get an extension to do this on save.

File: v2/manifests/core-ons/dp-feedback-api.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+services:
+  dp-feedback-api:
+    x-repo-url: "https://github.com/ONSdigital/dp-feedback-api"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-feedback-api
+      dockerfile: Dockerfile.local
```
Comment (lindenmckenzie) [inline v2/manifests/core-ons/dp-feedback-api.yml:6]: dp-feedback-api has no Dockerfile.local

File: v2/manifests/core-ons/dp-frontend-feedback-controller.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    ports:
+      - 25200:25200
+    restart: unless-stopped
+    environment:
+      BIND_ADDR:                          ":25200"
+    healthcheck:
+      test: ["CMD", "curl", "-sSf", "http://localhost:25200/health"]
+      interval: ${HEALTHCHECK_INTERVAL:-30s}
+      timeout: 10s
+      retries: 10
```
Comment (lindenmckenzie) [inline v2/manifests/core-ons/dp-frontend-feedback-controller.yml:26]: New line at EOF please - you can get an extension to do this on save.

**Review by lindenmckenzie — CHANGES_REQUESTED** [review]
Few bits of feedback - to properly review this though, it may be better to do one PR for the whole stack. As an example, there aren't many env vars here setting up the connections between services. This could be added later as part of the stack setup for example, but I'm not going to anticipate it in this review.

File: v2/manifests/core-ons/dp-feedback-api.yml (hunk 1 above)
Comment (dhee-tree) [reply v2/manifests/core-ons/dp-feedback-api.yml:12]: Added with this [PR](https://github.com/ONSdigital/dp-feedback-api/pull/16)

File: v2/manifests/core-ons/dp-feedback-api.yml (hunk 3 above)
Comment (dhee-tree) [reply v2/manifests/core-ons/dp-feedback-api.yml:6]: Added with this [PR](https://github.com/ONSdigital/dp-feedback-api/pull/16)

File: v2/manifests/core-ons/dp-frontend-feedback-controller.yml (hunk 4 above)
Comment (dhee-tree) [reply v2/manifests/core-ons/dp-frontend-feedback-controller.yml]: Done

**Pull request closed without merging**